modules/CmpdReg/src/client/custom/Lot_Custom.js

```diff
@@ -498,6 +507,9 @@ $(function() {
               if (this.purityMeasuredByCodeController.getSelectedModel().isNew()) purityMeasuredBy = null;
               else purityMeasuredBy = this.purityMeasuredByCodeController.getSelectedModel();
                 var vendor;
+              if (this.operatorCodeController.getSelectedModel().isNew()) purityOperator = null;
+              else purityOperator = this.operatorCodeController.getSelectedModel();
+                var tareWeightUnits; 
```

Comment:
Weird indentation here for `var tareWeightUnits` and then it is not used directly below. Was there supposed to be a `var purityOperator` above here?

Reply:
Thanks, fixed this.